independent-publisher-connectors/Google BigQuery/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+    "properties": {
+      "connectionParameters": {},

```

Comment:
@ashwanidv100 in your readme file you mentioned that users will need to generate an OAuth 2.0 client ID and secret for authentication. can you please set your connectionParameters properly.
Please see this IP connector [pull request](https://github.com/microsoft/PowerPlatformConnectors/pull/2087/files#diff-12c628f598d16514385609b53592f6242327ad5a84f04dab4b0ec41cdf07c2c1) for an example.

Reply:
Let me check on this.. I will confirm and fix.

Reply:
I think its updated now. In case this is still not correct, Please let me know.